jina/resources/health_check/pod.py

```diff
@@ -0,0 +1,39 @@
+import grpc
+
+from jina.serve.networking import GrpcConnectionPool
+from jina.types.request.control import ControlRequest
+
+
+def check_health_pod(addr: str):
+    """check if a pods is healthy
+
+    :param addr: the address on which the pod is serving ex : localhost:1234
+    """
+
+    try:
+        GrpcConnectionPool.send_request_sync(
+            request=ControlRequest('STATUS'),
+            target=addr,
+        )
+    except grpc.RpcError as e:
+        print("The pod is unhealthy")
```

Comment:
I can buy a `logger`, but isn't printing dirty?

Reply:
what would be the alternative, sys.stdout and sys.stderr ?

Reply:
a logger would work? Not sure, otherwise leave it like this

Reply:
yes but it is a bit overkill I think we can't even acces the output of this health check easily within docker compose